### DISCUSSION TITLE
Unable to join the Documentation Matrix Community

### DISCUSSION THREAD

**Discussion by bkp5190** [nix-community/nixvim#2504] [Q&A] [answered]
Has anyone else ran into an issue with joining the Matrix thread for the documentation of this repository?
<img width="780" alt="Screenshot 2024-11-11 at 9 04 55 AM" src="https://github.com/user-attachments/assets/a49971b3-50ba-44c7-9311-6a8c8afeef39">

**Comment by MattSturgeon** [comment, marked as answer]
Does this link work? https://matrix.to/#/#nixvim-documentation:matrix.org

What about if you join the "space" first? https://matrix.to/#/#nixvim:matrix.org

**Reply by bkp5190** [reply]
Resolution: Join the "space" first, then join the documentation room. Thanks Matt!
